src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -49,23 +49,24 @@
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(studies);
     }
 
-    @PostMapping(value = "/studies/{studyName}/cases/{caseName}")
+    @PostMapping(value = "/studies/{studyName}/cases/{caseUuid}")
     @ApiOperation(value = "create a study from an existing case")
     @ApiResponses(value = {
             @ApiResponse(code = 200, message = "The id of the network imported"),
             @ApiResponse(code = 409, message = "The study already exist or the case doesn't exists")})
     public ResponseEntity<Void> createStudyFromExistingCase(@PathVariable("studyName") String studyName,
-                                                                  @PathVariable("caseName") String caseName,
+                                                                  @PathVariable("caseUuid") UUID caseUuid,
                                                                   @RequestParam("description") String description) {
 
         if (studyService.studyExists(studyName)) {
             throw new ResponseStatusException(HttpStatus.CONFLICT, STUDY_ALREADY_EXISTS);
         }
 
-        if (!studyService.caseExists(caseName)) {
-            throw new ResponseStatusException(HttpStatus.NOT_FOUND, CASE_DOESNT_EXISTS);
+        if (!studyService.caseExists(caseUuid)) {
+            throw new ResponseStatusException(HttpStatus.CONFLICT, CASE_DOESNT_EXISTS);
```

Comment:
keep NOT_FOUND ?

Reply:
of course don't know what went through my mind...